script.js

```diff
@@ -1,4 +1,401 @@
+      `Submit "scissors", "paper", "stone", "reversed scissors", "reversed paper" or "reversed stone" to continue playing!<br ><br >` +
+      `You can also submit "reverse" again to reverse the game rules.`;
+    return reversedResult;
+  }
+};
+
+// ===== Function to pick an option of either scissors, paper or stone =====
+var pickOption = function () {
+  var randomOptionNum = Math.floor(Math.random() * 3); // Generate random integer between 0 to 3, exlucing 3
+  var selectedOption = "";
+  // Alternative way to random pick an option
+  // var optionsArray = ["scissors", "paper", "stone"]; // Set the output options in an array
+  // var selectedOption = optionsArray[randomOptionNum]; // Use the random integer to select the random item in the array
+  if (randomOptionNum == 0) {
+    selectedOption = "scissors";
+  } else if (randomOptionNum == 1) {
+    selectedOption = "paper";
+  } else {
+    selectedOption = "stone";
+  }
+  return selectedOption;
+};
+
+// ===== Helper functions to calculate winning percentage =====
+
+var calUserWinningPercent = function () {
+  var userWinPercent = (numOfTimesUserWon / numOfGamesPlayed) * 100;
+  return userWinPercent;
+};
+
+var calPcWinningPercent = function () {
+  var pcWinPercent = (numOfTimesPcWon / numOfGamesPlayed) * 100;
+  return pcWinPercent;
+};
+
+//  ===== Helper function to clear all counters to restart the Korean version =====
+
+var clearCounters = function () {
+  numOfTimesUserWon = 0;
+  numOfTimesPcWon = 0;
+  numOfDraws = 0;
+  numOfGamesPlayed = 0;
+  recentWinner = "";
+};
+
+//  ===== Function to determine winner =====
+
+var checkWhoWins = function (userSelection, computerSelection) {
+  var results = "";
+  numOfGamesPlayed = numOfGamesPlayed + 1;
+  // console.log(`Current game mode is ${gameMode}.`);
+  if (
+    userSelection == computerSelection ||
+    (userSelection == "reversed scissors" && computerSelection == "scissors") ||
+    (userSelection == "reversed paper" && computerSelection == "paper") ||
+    (userSelection == "reversed stone" && computerSelection == "stone")
+  ) {
+    numOfDraws = numOfDraws + 1;
+    results = "draw";
+  } else if (
+    (userSelection == "scissors" && computerSelection == "paper") ||
+    (userSelection == "paper" && computerSelection == "stone") ||
+    (userSelection == "stone" && computerSelection == "scissors") ||
+    (userSelection == "reversed scissors" && computerSelection == "stone") ||
+    (userSelection == "reversed paper" && computerSelection == "scissors") ||
+    (userSelection == "reversed stone" && computerSelection == "paper")
+  ) {
+    if (gameMode == "Original Mode") {
+      numOfTimesUserWon = numOfTimesUserWon + 1;
+      userWinningPercent = calUserWinningPercent().toFixed(0);
+      results = "user";
+    } else if (gameMode == "Reversed Mode") {
+      numOfTimesPcWon = numOfTimesPcWon + 1;
+      pcWinningPercent = calPcWinningPercent().toFixed(0);
+      results = "computer";
+    }
```

Comment:
Impressive job in combining both original and reversed that have the some outcome into the same statement!